src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -93,6 +94,20 @@ public void savesAndLoads() throws Exception {
         );
     }
 
+    @Test
+    public void saveOverwrites() throws Exception {
+        final byte[] original = "1".getBytes();
+        final byte[] updated = "2".getBytes();
+        Assume.assumeThat(updated, Matchers.not(Matchers.equalTo(original)));
+        final BlockingStorage storage = new BlockingStorage(
+            new FileStorage(this.folder.newFolder().toPath())
```

Comment:
@olegmoz I don't think that it is a good idea to share this between all tests

Reply:
@paulodamaso new folder is created for each new test, so the folder is not being reused

Reply:
@olegmoz Makes sense.